src/main/java/org/dimdev/dimdoors/pockets/SchematicV2Handler.java

```diff
@@ -58,9 +53,17 @@ public void load() {
                     result.add(line);
                 }
                 JsonObject json = GSON.fromJson(String.join("", result), JsonObject.class);
-                PocketType type = PocketType.CODEC.decode(JsonOps.INSTANCE, json).getOrThrow(false, System.err::println).getFirst();
-                this.pocketTypes.add(type);
-                this.loadResourceSchematics(type);
+                PocketGroup type = PocketGroup.CODEC.decode(JsonOps.INSTANCE, json).getOrThrow(false, System.err::println).getFirst();
+
+                this.pocketGroups.add(type);
+
+				WeightedList<VirtualPocket, PocketGenerationParameters> weightedPockets = new WeightedList<>();
```

Comment:
Indentation mismatch here

Reply:
Doesn't appear that way on my end, but I noticed that everything seems to have been indented using four spaces previously, while I indented everything using tabs. Not sure how to ensure correct whitespace type.

Reply:
Enable editorconfig in your ide and that should hopefully make everything use tabs